Tutorial_1b_Moment_calculation.ipynb

```diff
@@ -0,0 +1,860 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    #spec_noise_realistic: The sensitivity might decrease towards channesl at the edge, increasing the noise
</pre><p>Can you add a bit more backround here on why this occurs?</p><p> </p><p>Also typo on line 17</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>